data/sortOptions.json

```diff
@@ -0,0 +1,53 @@
+[
+    {
+        "id": "792fe292-fc9c-11ec-b939-0242ac120002",
+        "name": "Popularity",
+        "pathToValue": "stars",
```

Comment:
Given, `pathToValue` is always unique, why do we need a separate `id` we can simply use the `pathToValue` variable to signify both? 

Hard-coding random strings as IDs is not a great idea, and theoretically is not any different from using a unique readable string. I'd consider the latter better, owing to readability

Reply:
I have followed your suggestions and modified the code please review it again now